File: C/Program131.c
#include<stdio.h>

void UpdateLetter(char * str)
{
    
    while (*str != '\0')
    {
        if(*str == 'l')
        {
            *str = '-';
        }
        str++;
    }
}

int main()
{
    char Arr[50] = {'\0'};

    printf("Enter String:\n");
    scanf("%[^'\n']s",Arr);           

    UpdateLetter(Arr);
    
    printf("Updated String is: %s",Arr);

    return 0;
} 

File: Python/program131.py
def CountFrequency(Arr, iSize):
    iCount = 0

    for iCnt in range(iSize):
        if Arr[iCnt] == 11:
            iCount += 1

    return iCount


iLength = int(input("Enter the number of elements : "))

Brr = []

print("Enter the elements :")
for iCnt in range(iLength):
    Brr.append(int(input()))

iRet = CountFrequency(Brr, iLength)

print("Frequency :", iRet)
